src/http_server.cpp

```diff
@@ -315,6 +322,10 @@ namespace sim
 		m_recv_buffer.erase(m_recv_buffer.begin(), m_recv_buffer.begin() + req_len);
 		m_bytes_used -= req_len;
 
+		if (m_send_buffer.empty()) {
```

Comment:
I was thinking something like either a separate map, separate from the existing path -> handler one